### DISCUSSION TITLE
Is it possible to filter & modify requests using wiremock captain?

### DISCUSSION THREAD

**Discussion by mat-sharp** [HBOCodeLabs/wiremock-captain#324] [General]
The wiremock documentation describes how to extend stubRequestFilter in Java.  Is it possible to do that kind of action in wiremock-captain please?

**Comment by satvik-s** [comment]
hi! sorry for the late response as i totally missed this due to not getting tagged

i was looked at the wiremock documentation for the same here: https://wiremock.org/docs/extensibility/filtering-requests/

but did not find a JSON equivalent example provided that we can re-create to utilize it. as wiremock-captain relies on using the JSON equivalent documentation/implementation provided natively by wiremock, any feature that does not have JSON implementation support from wiremock won't be possible to implement in wiremock-captain

@mat-sharp
